src/app/clusters/groupcast-server/GroupcastServer.cpp

```diff
@@ -0,0 +1,105 @@
+#include "GroupcastServer.h"
+
+
+using chip::Protocols::InteractionModel::Status;
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace {
+static GroupcastCluster sInstance;
+} // namespace
+
+GroupcastCluster & GroupcastCluster::Instance()
+{
+    return sInstance;
+}
+
+GroupcastCluster::GroupcastCluster() : DefaultServerCluster({ kRootEndpointId, Groupcast::Id })
+{
+}
+
+CHIP_ERROR GroupcastCluster::Startup(ServerClusterContext & context)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
+
+void GroupcastCluster::Shutdown() {}
```

Comment:
same here: please derive DefaultServerCluster and use the defaults there. Do not override them (and if you do, you must call the base class method since they do stuff)

Reply:
Removed.